docs/_docs/persistence/change-data-capture.adoc

```diff
@@ -171,6 +171,24 @@ For example, CDC was turned off several times: `000000000000002.wal`, `000000000
 Then, after the command is executed, the following segment links will be deleted: `000000000000002.wal`, `000000000000003.wal`, `000000000000008.wal`.
 The application will start from the `0000000000000010.wal` segment after being enabled.
 
+== Flush existing caches
+
+When the CDC has been forcefully disabled for a while, cache changes will be skipped.
+In this case it is necessary to flush data from existing caches. For example, this is important for the following cases:
+
+- pre-synchronizing replication;
+- handling missed segments.
+
+To flush cache entries you can run the following link:tools/control-script[Control Script] command:
+
+[source,shell]
+----
+# Flush cache data entries. Iterates over caches and writes data entries to the WAL to get captured by CDC:
+control.sh|bat --cdc flush_caches --caches cache1,...,cacheN
+----
+
+The command will iterate over caches and writes data entries to the WAL to get captured by CDC application.
```

Comment:
Please, add information about handling concurrent updates.
Looks like we doesn't provide any guarantees, so we must state this explicitly in documentation.

Reply:
Fixed